frontend/ui/widget/infomessage.lua

```diff
@@ -196,6 +196,9 @@ function InfoMessage:init()
 end
 
 function InfoMessage:onCloseWidget()
+    if self._delayed_show_action then
+        UIManager:unschedule(self._delayed_show_action)
+    end
```

Comment:
Might do here and below in:
```diff
function InfoMessage:dismiss()
    if self._delayed_show_action then
        UIManager:unschedule(self._delayed_show_action)
+       self._delayed_show_action = nil
```
to avoid UIManager:unschedule twice, even if the 2nd is a no-op ?

Reply:
Yeah, good point!